manifests/o/Opera/Opera/117.0.5408.35/Opera.Opera.locale.nb-NO.yaml

```
@@ -0,0 +1,20 @@
+PackageIdentifier: Opera.Opera
+PackageVersion: 117.0.5408.35
+PackageLocale: nb-NO
+Publisher: Opera Software
+PublisherUrl: https://www.opera.com/no
+PublisherSupportUrl: https://www.opera.com/no/help
+PrivacyUrl: https://legal.opera.com/privacy/
+Author: Opera Software
+PackageName: Opera Stable
+PackageUrl: https://www.opera.com/no/browsers/opera

```

My suggestion:

```suggestion
PackageUrl: https://www.opera.com/browsers/opera
```

It (at least in mainland China) automatically jumps to pages in each language.